build.psm1

```diff
@@ -914,9 +964,28 @@ Make sure that the proper git submodules are installed by running:
         $outputBufferFilePath = [System.IO.Path]::GetTempFileName()
     }
 
-    $Command += "Invoke-Pester "
+    $testPaths = @{}
+    if ($UseRunspacePool)
+    {
+        # Find all directories that contain .Tests.ps1 files
+        $testScriptPaths = @()
+        foreach ($subPath in $Path)
+        {
+            $testScriptPaths += (Get-ChildItem -Recurse -Path $subPath -Filter *.?ests.ps1 | Select-Object -Property DirectoryName -Unique).DirectoryName
+        }
+        # Remove any child paths included in parents
+        $prevPath = (Get-Random).ToString()
+        foreach ($testPath in $testScriptPaths)
+        {
+            if (!$testPath.Contains($prevPath))
```

Comment:
Can we filter for Remoting folder here? Maybe move the filtering to a function so we can easily add more folders to be filtered if necessary.